### PR TITLE
Email Subscriptions: Create DeliveryWindowInput component

### LINKED ISSUE #74470: `/subscriptions/settings`: Create “Daily/weekly delivery window” input
## Design
inDLaEQV8jJ21O4WXawIsJ-fi-712%3A29253

### Mobile Design
Image

### Desktop Design
Image

## Acceptance Criteria
- Display the setting as per the Figma design
- Have the relevant, passed-in user setting props reflected as the value of the inputs
- Ensure any displayed text, that needs translation, is submitted for translation
- **Note:** There is no need to deal with the submission of the changed value as part of this task. The submission will be implemented with the following task, once all the settings form inputs are being included in the parent component: https://github.com/Automattic/wp-calypso/issues/74433

## Related Conversations
- p1678792364571509-slack-C02TCEHP3HA
- p1679328752556829-slack-C02TCEHP3HA

## Dev Notes
- The exact same setting, but for a logged-in user, is implemented in other part of Calypso: http://calypso.localhost:3000/me/notifications/subscriptions
   Image

### PR DESCRIPTION
**Pull request by phcp**
Resolves #74470

## Proposed Changes

* Creates DeliveryWindowInput component

## Testing Instructions

* Deploy this PR on your local env
* Open http://calypso.localhost:3000/subscriptions/settings
* Verify if the page is rendered with the Delivery Window field

Screenshot 2023-03-17 at 16 17 54

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+		.tip-icon {
+			background: $studio-white $tip_icon_base64 no-repeat center;
+			height: 20px;
+			width: 20px;
+		}
+
+		.tip-text {
+			color: $studio-gray-40;
+			font-size: 0.875rem;
```
Comment (mashikag) [inline packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss]: We should use our predefined SCSS constants whenever possible. For font-sizes and line heights, we have those defined in `@automattic/typography`.
 
Screen Shot 2023-03-17 at 20 54 17
```suggestion
			font-size: $font-body-small;
```

File: packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			font-weight: 400;
+			line-height: 20px;
```
Comment (mashikag) [inline packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss]:
```suggestion
			line-height: $font-title-small;
```

**Comment by JuanLucha** [comment]
We have a mismatch in font sizes:
- Design:
Screenshot 2023-03-20 at 13 14 09

- Your implementation:
Screenshot 2023-03-20 at 13 13 57

As a note, the same font mismatch can be found in another part of the design:
Screenshot 2023-03-20 at 13 18 00

File: packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss (hunk 1 above)
Comment (phcp) [reply packages/subscription-manager/src/components/fields/DeliveryWindowInput/styles.scss]: Fixed!

**Review by JuanLucha — APPROVED** [review]
Good job! 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7878905

Thank you @phcp for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7878905).

**Pull request merged**